helm/grafana/templates/grafana-nginx-configmap.yaml

```diff
@@ -0,0 +1,38 @@
+{{- if .Values.grafanaProxy.enabled }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: grafana-nginx-config
+  namespace: {{ .Release.Namespace }}
+  labels: {{- include "labels.selector" . | nindent 4 }}
```

Comment:
Please make sure resource metadata is consistent across the whole chart.
For instance here are a few guidelines I'm thinking about:
- common labels
- name should include `{{ .Chart.Name }}` to be unique
- it should not try to define the helm-defined annotations

This remark is not only for the `grafana-nginx-config` configmap, but for all of the resources added in this PR.